src/pages/ConsumerPurposeCreatePage/components/PurposeCreateForm.tsx

```diff
@@ -179,7 +178,10 @@ export const PurposeCreateForm: React.FC = () => {
               {isProviderPurposeSelected && (
```

Comment:
```suggestion
              {isProviderPurposeSelected &&
                selectedProviderRiskAnalysisId &&
                selectedProviderRiskAnalysisId && (
```
should be better if we add a check here and don't force 'as string' on `eserviceId` and riskAnalysisId props